src/blame.c

```diff
@@ -133,8 +133,11 @@ git_blame* git_blame__alloc(
 		return NULL;
 	}
 
-	if (opts.flags & GIT_BLAME_USE_MAILMAP)
-		git_mailmap_from_repository(&gbr->mailmap, repo);
```

Comment:
I think ignoring the error was actually intended. In case where we're unable to load the mail map, we simply ignore it and output commits with their original author/committer. You could add a comment and cast to `void`, though

Reply:
Ignore the error even though I (the user) has asked for it ? I do not know if that's explicit, but I find this strange… How can I tell if the mailmap was applied when that happens ?